ospf6d/ospf6_abr.c

```diff
@@ -53,6 +53,16 @@
 
 unsigned char conf_debug_ospf6_abr;
 
+int ospf6_ls_origin_cmp(struct ospf6_path *o_path, struct ospf6_route *route)
```

Comment:
Let's please rename this to `_same` or swap the return value.
The `_cmp` family usually returns zero when arguments are equal.

Reply:
Done. Update the function name to ospf6_ls_origin_same.